packages/types/index.d.ts

```diff
@@ -221,6 +221,7 @@ export interface IFieldDropdownSource {
 export interface IFieldDropdownOption {
   label: string
   value: boolean | string | number
+  description?: string
```

Comment:
can add comment to not put both `value` and ` description` by accident

Reply:
niceeee